### ISSUE TITLE
[Bug][Accessibility] iOS CollectionView with VoiceOver not usable in Xamarin.Forms 4.8

### ISSUE THREAD

**Issue by aryobenholzner** [xamarin/Xamarin.Forms#12252]
### Description
In Xamarin.Forms 4.8, when using VoiceOver, the contents of a CollectionView do not get accessible and thus, items cannot be selected

### Steps to Reproduce
see repro project
1. add a CollectionView with a few items and SelectionMode="Single"
2. deploy to an iOS device
3. turn VoiceOver on
4. try navigating through the CollectionView

### Expected Behavior
The CollectionView is accessible by VoiceOver and items can be selected

### Actual Behavior
The CollectionView cannot be accessed by voiceOver. The user is unable to navigate through the items or select them

### Basic Information

- Version with issue: 4.8.0.1451
- Last known good version: 4.7.0.1080
- Platform Target Frameworks: 
  - iOS:  13.7
- Affected Devices: iOS devices

### Reproduction Link
[collectionview.zip](https://github.com/xamarin/Xamarin.Forms/files/5282037/collectionview.zip)

**Comment by jameslavery** [comment]
@aryobenholzner  - one workaround (which may not be useful for you, depending on your use case) is to set

_AutomationProperties.IsInAccessibleTree="False"_ on the CollectionView

This allows you to select the items. This is what we've had to do for now in our app.

**Comment by rachelkang** [comment]
Hi all, this issue has since been fixed!

If you haven't already, I recommend updating to the latest version Xamarin.Forms or at least to Xamarin.Forms 5.0.0.2083, which has many accessibility improvements. These improvements include an experimental accessibility flag that addresses a lot of improvements in Android accessibility, and other areas like this iOS CollectionView one as well.

To leverage this flag on the latest versions of Xamarin Forms, the following code can be inserted in the App.xaml.cs file, following the InitializeComponent call and before the MainPage is set: `Xamarin.Forms.Device.SetFlags(new List<string> { "Accessibility_Experimental"});`

Let us know how that works for you!
